certified-connectors/Microsoft Bookings/apiDefinition.swagger.json

```diff
@@ -266,7 +266,7 @@
                     "Deleted"
                   ],
                   "items": {
-                  "type": "string"
+                    "type": "string"
```

Comment:
@vikailas please fix the indentation here.

Reply:
Resolved.